pyrelational/pipeline/pipeline.py

```diff
@@ -264,7 +264,7 @@ def test_loader(self) -> DataLoader[Any]:
         return self.data_manager.get_test_loader()
 
     @property
-    def get_percentage_labelled(self) -> float:
```

Comment:
When did we revert that back? I don't mind really, but just recalled adding the `get_` prefix to this.

Reply:
I revert it back in this PR because it's not technically a getter here, it's a property, so it's the actual percentage. It's really a nitpick from me...